shared/bash_remediation_functions/disable_prelink.sh

```diff
@@ -0,0 +1,16 @@
+function disable_prelink {
+	# Disable prelinking and don't even check
+	# whether it is installed.
+	if grep -q ^PRELINKING /etc/sysconfig/prelink
+	then
+		sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
+	else
+		printf '\n%s\n' '# Set PRELINKING=no per security requirements' >> /etc/sysconfig/prelink
```

Comment:
why not add both in one statement? why printf then echo?

Reply:
also, maybe I am missing something but I don't get the point of the `printf`, it's just putting a string in a string

Reply:
There used to be `echo -e`, which sucks. Printf is safer, but I have rewritten it, so it provokes less.